apis/core/lib/domain/table/Table.ts

```diff
@@ -92,6 +92,7 @@ export default function mixin<Base extends Constructor<Table>>(Resource: Base):
       return this.addLiteralColumnMapping({
         store,
         sourceColumn: column,
+        datatype: column.defaultDatatype,
```

Comment:
More importantly though, I wonder if we need this as a field of `CsvColumn`. Maybe just call here?

```suggestion
        datatype: inferDatatype(column.samples),
```

Reply:
good observation, done